packages/calypso-e2e/src/lib/pages/reader-page.ts

```diff
@@ -35,6 +38,25 @@ export class ReaderPage {
 		await this.page.waitForLoadState( 'load' );
 	}
 
+	/**
+	 * Searches the Reader stream with a keyword.
+	 *
+	 * @param {string} keyword Keyword to search.
+	 */
+	async search( keyword: string ): Promise< void > {
+		await Promise.all( [
+			this.page.waitForNavigation(),
```

Comment:
Does this have any effect here? I'm asking because it doesn't seem like it's redirecting, only setting query params to the URL. Maybe using the `waitForResponse` would be a better match? Something like:

```suggestion
			this.page.waitForResponse( /read\/search/ ),
```

Reply:
Playwright treats this as a navigation. See the following snippet generated with `DEBUG=pw:api` set:

```
  pw:api   navigated to "https://wordpress.com/read/search?q=&focus=1" +164ms
  pw:api   "load" event fired +281ms
  pw:api   "networkidle" event fired +4s
  pw:api   navigated to "https://wordpress.com/read/search?q=Test+String&focus=1" +94ms
```

Reply:
Thanks for the snippet. Looks like every change of the URL is considered a navigation change, then! Good to know.